dhis-2/dhis-test-e2e/src/test/java/org/hisp/dhis/tracker/imports/events/EventsDataValueValidationTests.java

```diff
@@ -126,11 +126,7 @@ public void shouldValidateWhenNoDataValue(String validationStrategy, String even
   }
 
   @ParameterizedTest
-  @CsvSource({
-    "ON_COMPLETE,ACTIVE",
-    "ON_UPDATE_AND_INSERT,SCHEDULE",
-    "ON_UPDATE_AND_INSERT,SKIPPED"
-  })
+  @CsvSource({"ON_COMPLETE,ACTIVE"})
```

Comment:
What does this do?

Reply:
This test was checking the that mandatory values in `SCHEDULE` and `SKIPPED` statuses were not validated but now we are even more restrictive so those tests cannot be reproduced anymore.